onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlSTFT.h

```diff
@@ -0,0 +1,610 @@
+#pragma once
+
+#include "DmlDFT.h"
+
+enum DmlSTFTKernelInputIndex : uint32_t
+{
+    Signal = 0,
```

Comment:
> = 0 [](http://example.com/codeflow?start=10&length=4)

[minor] Don't C++ enums always start at zero anyway? (unless you explicitly say otherwise)